### PR TITLE
add gkv yaml file

### PR DESCRIPTION
**Pull request by ybur-yug**
Let me know if the tags could be better. I wanted to do 'kv store' but wasnt sure if multiword tags worked. Thanks!

### CODE REVIEW

File: _data/projects/gkv.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/ybur-yug/gkv
+tags:
+  - oss
+  - databases
+  - ruby
+  - db
+  - kv
+upforgrabs:
+  name: Gkv
+  link: https://github.com/ybur-yug/gkv
```
Comment (shiftkey) [inline _data/projects/gkv.yml]: This link should be to the list of issues which are recommended for new contributors, e.g:

https://github.com/ybur-yug/gkv/labels/help%20wanted

`help wanted` then becomes the `name` value above...

File: _data/projects/gkv.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Gkv
+desc: An open source, git-based key value store for small, simple projects
+site: https://github.com/ybur-yug/gkv
+tags:
+  - oss
+  - databases
+  - ruby
+  - db
+  - kv
```
Comment (shiftkey) [inline _data/projects/gkv.yml]: Perhaps don't worry about these acronyms. 

As you're using `git` here in an interesting way, perhaps call that out.

**Comment by ybur-yug** [comment]
I changed it up per your suggestions @shiftkey 

Better?

**Comment by dahlbyk** [comment]
I see you adjusted the link to point to your roadmap, but I'd suggest actually creating labeled issues to represent the work that needs to be done. The folks we're targetting here are likely to need more direction than a bullet point to encourage their contribution to your project.

**Comment by ybur-yug** [comment]
Got it. I can open that up now and will update the link. Thanks @dahlbyk

**Comment by ybur-yug** [comment]
I've added issues for the two main features to be added in 0.3

Is this about what you were looking for? While there are many possible solutions I just made sure to clarify "if it works, it will be accepted" as I am not too stingy and the project is so young.

Sorry if I'm misfiring here. Let me know if this is what was desired.

Thanks

**Comment by ybur-yug** [comment]
PS sorry I accidentally hit close and comment :stuck_out_tongue_winking_eye:

File: _data/projects/gkv.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: Gkv
+desc: An open source, key value store for small, simple projects leveraging git as the storage engine
+site: https://github.com/ybur-yug/gkv
+tags:
+  - database
+  - ruby
+  - git
+upforgrabs:
+  name: Gkv
+  link: https://github.com/ybur-yug/gkv/issues
```
Comment (shiftkey) [inline _data/projects/gkv.yml:10]: This should be a URL to the filtered list of issues curated for new contributors.

I can't see any particular labels which might suit this, but I'd suggest `help wanted`, which maps to this URL: https://github.com/ybur-yug/gkv/labels/help+wanted

File: _data/projects/gkv.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+name: Gkv
+desc: An open source, key value store for small, simple projects leveraging git as the storage engine
+site: https://github.com/ybur-yug/gkv
+tags:
+  - database
+  - ruby
+  - git
+upforgrabs:
+  name: Gkv
```
Comment (shiftkey) [inline _data/projects/gkv.yml:9]: Similarly, this is the label name associated with the curated issues - `help wanted` is the example from the default repository.

**Comment by shiftkey** [comment]
Closing this out as inactive. Feel free to open a fresh PR if you'd still like to see your project listed.

**Pull request closed without merging**